rest/route/host_agent.go

```diff
@@ -1095,11 +1095,13 @@ func sendBackRunningTask(ctx context.Context, env evergreen.Environment, h *host
 	}
 	if t == nil {
 		grip.Notice(getMessage("clearing host's running task because it does not exist"))
+		runningTask := h.RunningTask
```

Comment:
Can we add a comment for why this fixes a bug so that someone doesn't think it's redundant in the future?